DetectorDescription/DDCMS/src/DDFilteredView.cc

```diff
@@ -19,6 +27,25 @@ const Double_t* DDFilteredView::trans() const { return it_.back().GetCurrentMatr
 
 const Double_t* DDFilteredView::rot() const { return it_.back().GetCurrentMatrix()->GetRotationMatrix(); }
 
+void DDFilteredView::rot(dd4hep::Rotation3D& matrixOut) const {
```

Comment:
This method is called by the MF builder code to provide the rotation as a clean `Rotation3D` object, rather than an array that could be misinterpreted. I'm not sure if I'm answering your question.